.github/workflows/gradle.yml

```diff
@@ -0,0 +1,20 @@
+name: Gradle Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Build with Gradle
+      run: ./gradlew -Pversion=$GITHUB_REF release --scan --no-daemon -i
```

Comment:
IIRC we need to explicitly opt-in for scans

Reply:
What do you mean, precisely? We have `--scan` here (as we did for Travis), and `build.gradle` includes the flag for acceptance of the terms of service. Is there something I've missed?

Reply:
ooops, sorry, my bad, I though we ask to opt-in :) Good then 👍